mParticle-Tune/MPKitTune.m

```diff
@@ -65,11 +81,11 @@ - (instancetype)initWithConfiguration:(NSDictionary *)configuration startImmedia
     }
 
     _platform = @"ios";
-    MPDevice *device = [[MPDevice alloc] init];
-    _identifierForAdvertiser = device.advertiserId;
-    MPApplication *application = [[MPApplication alloc] init];
+    _identifierForAdvertiser = [self advertiserId];
+    NSDictionary *bundleInfoDictionary = [[NSBundle mainBundle] infoDictionary];
+    NSString *bundleIdentifier = bundleInfoDictionary[@"CFBundleIdentifier"];
```

Comment:
`bundleInfoDictionary[@"CFBundleIdentifier"]` can return `nil`. Is the code okay with that?